Replace term scanner to finder and improve text (#244)
encukou/freezeyt: README.md

### URL finding

> `freezeyt` discovers new links in the application by url finders. Url finders

Reviewer: ```suggestion
`freezeyt` discovers new links in the application by URL finders. URL finders
```

are functions whose goal is to find url of specific MIME type.

> `freezeyt` offer different configuration options to use url finders:

Reviewer: ```suggestion
`freezeyt` offer different configuration options to use URL finders:
```

> * use predefined url finders for `text/html` or `text/css` (default),
> * define your own url finder as your function,

Reviewer: ```suggestion
* use predefined URL finders for `text/html` or `text/css` (default),
* define your own URL finder as your function,
```

* turn off some of finders (section below)

Example of configuration:

```yaml
url_finders:
    text/html: get_html_links
    text/css: get_css_links
```

Keys in the `url_finders` dict are MIME types;

> Values are url finder, which can be define as:

Reviewer: ```suggestion
Values are URL finders, which can be defined as:
```

* strings in the form `"module:function"`, which name the finder
  function to call,
* strings like `get_html_links`, which name a function from the
  `freezeyt.url_finders` module, or
* Python functions (if configuring `freezeyt` from Python rather than
  YAML).

> If you'd like to define your url finder consider the right interface.

Reviewer: ```suggestion
```

> Input args:

Reviewer: ```suggestion
An URL finder gets these arguments:
```

* page content `BinaryIO`,
* the URL of the page, as a `string`,
* the HTTP headers, as a list of tuples (WSGI).

> Output:

Reviewer: ```suggestion
```

The function should return an iterator of URLs (as strings) found
in the page's contents. These URLs can be relative.

The `freezeyt.url_finders` module includes the default finders `get_html_links`
and `get_css_links`.

#### Disabling default URL finders

> If finder is not explictly specified in configuration file, `freezeyt` use the
> default for certain MIME type. For example, if you specify
> `text/html: my_custom_finder` only, `freezeyt` use for `text/css`
> the default finder.

Reviewer: ```suggestion
If a finder is not explictly specified in the configuration file, `freezeyt` will use the
default for certain MIME type. For example, if you specify
`text/html: my_custom_finder` only, `freezeyt` will use the default finder
for `text/css`.
```

> It's needed to explicitly disable this behaviour:

Reviewer: ```suggestion
You can disable this behaviour:
```
